packages/edit-post/src/components/fullscreen-mode/style.scss

```diff
@@ -1,4 +1,4 @@
-body.is-fullscreen-mode {
+body.is-fullscreen-mode:not(.no-js) {
```

Comment:
rather than `:not(.no-js)` I'd simply use `body.js.is-fullscreen-mode`

Reply:
Thank you @afercia !